include/octo-logger-cpp/context-info.hpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file context-info.hpp
```

Comment:
Please move the definitions to a src file.

Reply:
done